frontend/src/pages/article/components/article-form/article-form.tsx

```diff
@@ -49,22 +49,31 @@ const ArticleForm: React.FC<Properties> = ({ articleForUpdate }) => {
   const handleArticleSubmit = useCallback(
     (articleSubmitType: ValueOf<typeof ArticleSubmitType>) =>
       (payload: ArticleRequestDto): void => {
+        const isArticlePublish =
+          articleSubmitType === ArticleSubmitType.PUBLISH;
+
         const updatedPayload = {
           ...payload,
-          publishedAt:
-            articleSubmitType === ArticleSubmitType.PUBLISH
-              ? new Date().toISOString()
-              : null,
+          publishedAt: isArticlePublish ? new Date().toISOString() : null,
         };
 
         void dispatch(
           articlesActions.createArticle({
             articlePayload: updatedPayload,
             generatedPrompt: getGeneratedPromptPayload(generatedPrompt),
           }),
-        );
+        )
+          .unwrap()
+          .then(() => {
+            navigate(
+              isArticlePublish
+                ? ArticleSubRoute.ARTICLES
+                : ArticleSubRoute.MY_ARTICLES,
+            );
+          })
+          .catch(() => {});
```

Comment:
Do I understand right that in case of an error we do not see anything?

Reply:
To be honest, I don't know why I added it, I just merged development branch and there was the same logic so I made it the same way)